flask_resty/pagination.py

```diff
@@ -542,10 +560,28 @@ def get_filter_clause(self, column_cursors):
             value = int(value)
 
         if asc:
-            current_clause = column > value
+            if value is None:
+                return None
+            elif value is not None:
+                current_clause = self._handle_nullable(
+                    column, value, is_nullable
+                )
+            else:
+                current_clause = column > value
         else:
-            current_clause = column < value
+            if value is None:
+                current_clause = column.isnot(None)
```

Comment:
why does the descending case need different logic for null handling?

Reply:
For DESC NULLS came first, so we need to check if there are any objects that have any value example
```
SELECT * FROM widgets
WHERE widget.name IS NULL AND widget.id > 3 OR widget.name IS NOT NULL

```